veg2hab/habitat.py

```diff
@@ -346,18 +356,18 @@ def try_to_determine_habkeuze(
                         status=KeuzeStatus.HABITATTYPE_TOEGEKEND,
                         habtype=true_voorstellen[0].habtype,
                         kwaliteit=true_voorstellen[0].kwaliteit,
-                        habitatvoorstellen=true_voorstellen,
+                        habitatvoorstellen=all_voorstellen,
```

Comment:
Gaat deze goed, worden de habitatvoorstellen in de habitatkeuze verderop nog gebruikt voor iets?

Reply:
Enkel voor het geven van de output. Voor het checken van volgende iteraties mozaiekregels enzo worden de voorstellen uit de HabitatVoorstellen kolom gebruikt.